src/classes/ADDR_Account_TDTM.cls

```diff
@@ -176,14 +176,15 @@ public class ADDR_Account_TDTM extends TDTM_Runnable {
         list<Address__c> listAddr = new list<Address__c>();
         for (Account acc : listAcc) {
             Address__c addr = new Address__c();
-            addr.Household_Account__c = acc.Id;
-            addr.Default_Address__c = true;
-            addr.Latest_Start_Date__c = system.today();
-            addr.Latest_End_Date__c = null;
-            ADDR_Addresses_TDTM.copyAddressStdSObjAddr(acc, 'Billing', addr, null);
-            listAddr.add(addr);         
+            NPSP_Address npspAddress = new NPSP_Address(addr);
```

Comment:
If we eventually want to move this to an ORM, the calls to `getRecord().{fieldname}` would need to be replaced with something like `.setAccount(acc.id)`, etc. In this example though, I'm not sure we're really gaining any value by instantiating `NPSP_Addresses` in this if-block context. It's still directly manipulating the field instances and in the end we just add the record to the listAddr collection and the instance of NPSP_Address goes away at the curly brace on line 185. It honestly might be more efficient to stick with the original code here, unless we're going to implement a full ORM at this point - but that would require a lot more changes across this class.

Reply:
More logic has been moved into the NPSP_Address class now, including the rest of this block.  The getRecord() methods will need to stay for a while, until we get all appropriate properties exposed as public methods, at that point we'll be able to make getRecord() private.